LPPC/Plugins/topsky/TopSkySettings.txt

```diff
@@ -75,261 +67,169 @@ CPDLC_MinLevel=100
 CPDLC_Transfer=1
 CPDLC_AutoAcceptLogon=1
 CPDLC_AutoAcceptLogon_Time=15
-CPDLC_PlaySound=1
-CPDLC_PDC_PlaySound=1
+CPDLC_PDC_Auto=1
 CPDLC_PDC_Timeout_Pilot=180
 CPDLC_FSM_CLD_Header=CLD <hour><min> <year2><month2><day> <adep2> PDC <number> <callsign>
 
 Divergence_CLAM_DelayTime=30
 Divergence_RAM_Limit=1
 
-FlightLeg_Label_Time=1
+Equip_UseCodes=3
+
+FlightLeg_AssumeTime=2
+FlightLeg_AutoShow=1
 FlightLeg_Label_Point=0
 FlightLeg_Label_DIST=0
 FlightLeg_Label_RFL=0
 FlightLeg_Label_SCFL=0
-FlightLeg_AutoShow=1
-FlightLeg_AssumeTime=2
-FlightLeg_LineWidth=1
+FlightLeg_Potential_Risk=0
 
 FPCP_MinState=0
 
-HTTP_SIGMET_Pages=6443,6450
+HTTP_GRIB_Latitude_Max=45
+HTTP_GRIB_Latitude_Min=30
+HTTP_GRIB_Longitude_Max=-5
+HTTP_GRIB_Longitude_Min=-20
 
 HTTP_METAR_Server=https://meteo.vatsim.pt/
 HTTP_METAR_Page_Prefix=api/v1/ipma/metar/topsky?airport_icao_codes=
 HTTP_METAR_Page_Suffix=
 HTTP_METAR_Data_Prefix=(prefix)
 HTTP_METAR_Data_Suffix=(suffix)
 
+HTTP_natTrak_Filter_ADEP_Lon_Min=-22
+
 HTTP_NOTAM_Area_Sched_Text= 
 HTTP_NOTAM_Server=https://eaup.vatsim.pt/api/v1/topsky/areas/LP/notam/
 HTTP_NOTAM_Page_Prefix=?loc=
 HTTP_NOTAM_Page_Suffix=
 
-HTTP_TAF_Server=https://aviationweather.gov/
-HTTP_TAF_Page_Prefix=cgi-bin/data/dataserver.php?requestType=retrieve&dataSource=tafs&stationString=
-HTTP_TAF_Page_Suffix=&hoursBeforeNow=1&format=xml&mostRecentForEachStation=constraint&fields=raw_text
-HTTP_TAF_Data_Prefix=<raw_text>
-HTTP_TAF_Data_Suffix=</raw_text>
-
-Label_All_N=1
+Label_ACF_Use=2
```

Comment:
```suggestion
Label_ACF_Use=0
```
por testar

Reply:
Tenho duvidas de como será o real
0 é como temos atualmente na versão antiga do topsky

Reply:
Tenho a certeza absoluta que deve mostrar CA ou wtv no CFL, a minha unica duvida e so porque nao testei que era esse o comportamento do setting

Reply:
how can you be so sure?
Não há ref concreta disso
Mas o setting é esse para o comportamento que queres